newdle/client/src/components/CreateNewdle.module.scss

```diff
@@ -1,5 +1,65 @@
+@import '../styles/palette.scss';
+
 .button-row {
   display: flex;
   justify-content: flex-end;
   width: 100%;
 }
+
+.input-title {
+  color: $purple !important;
+}
+
+.title-input {
+  width: 100%;
+}
+
+.attention-message {
+  border: 1px solid $coral;
+  border-radius: 3px;
+  padding: 15px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  color: $grey;
+
+  .header {
+    color: $coral !important;
+  }
+}
+
+.create-button {
+  display: flex;
+  justify-content: center;
+}
+
+.success-message {
+  border: 1px solid $light-teal;
+  border-radius: 3px;
+  padding: 15px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  color: $grey;
+
+  .header {
+    color: $light-teal !important;
+  }
+}
+
+.summary-button {
+  display: flex;
+  justify-content: center;
+}
+
+.newdle-title {
+  display: flex;
+  justify-content: center;
+  color: $purple !important;
+}
+
+.newdle-link {
+  display: flex;
+  justify-content: center;
+  padding: 0.67857143em 1em;
```

Comment:
where is this weird value coming from?

Reply:
I wanted to use exactly the same padding semantic uses for inputs but I can change it to a less weird value.

Reply:
no in that case it's fine (but maybe add a comment mentioning that)